otx/cli/tools/optimize.py

```diff
@@ -118,12 +131,28 @@ def main():
     dataset_class = get_dataset_class(template.task_type)
 
     # Create instances of Task, ConfigurableParameters and Dataset.
+    train_ann_files, train_data_roots = args.train_ann_files, args.train_data_roots
```

Comment:
Same here with eval.py. If someone wants to export another dataset, not in data.yaml?

Reply:
Corrected and explained the same in the comments above.